mix.exs

```diff
@@ -1,19 +1,20 @@
 defmodule ADBC.MixProject do
   use Mix.Project
 
+  @app :adbc
   @version "0.1.0"
   @github_url "https://github.com/cocoa-xu/adbc"
 
   def project do
     [
-      app: :adbc,
+      app: @app,
       version: @version,
       elixir: "~> 1.12",
       start_permanent: Mix.env() == :prod,
       deps: deps(),
       package: package(),
       docs: docs(),
-      compilers: [:elixir_make] ++ Mix.compilers(),
+      compilers: [:elixir_make] ++ Mix.compilers() ++ [:adbc_driver],
```

Comment:
```suggestion
      compilers: [:elixir_make] ++ Mix.compilers(),
```

As per above, I don't really think we need a compiler. Originally I thought we did, but we don't. :)